woocommerce/woocommerce-android
# [Hack Week] Allow fixing cha-ching sound issue from settings

Closes: #10327 


### Description
This PR is the last of the Hack Week work, it adds a notifications screen that's used only when the cha-ching sound is missing, and have a button that allows fixing the issue, this will be useful in case the user dismissed the alert of #10336 accidenally.

### Testing instructions
1. Open the app information screen on your device, then go to the Notifications menu, then update the "New Order" category to make it silent.
2. Open the app, then open the settings.
3. Notice the notifications entry says "Notifications" now.
4. Click on it, and notice it opens a separate screen.
5. Click on "Manage notifications" and confirm it opens the device's app notifications screen.
6. Click on "Enable cha-ching sound"
8. Click on the "Test sound" button of the snackbar.
9. Confirm a test notification is posted, and uses the Cha-ching sound.
10. Go back to the app information screen, and check that the "New Order" category has been re-created and has the correct sound now.
11. Optional: test a push notification, and confirm it uses the Cha-ching sound.

### Images/gif
<img width=360 src="https://github.com/woocommerce/woocommerce-android/assets/1657201/eb6da232-52d2-4e97-80ac-05988052af5a"/>
